apps/src/templates/progress/StageExtrasProgressBubble.jsx

```diff
@@ -41,14 +46,29 @@ class StageExtrasProgressBubble extends Component {
       <a
         href={stageExtrasUrl + currentLocation().search}
         style={{
-          ...styles.main,
-          ...(onStageExtras && styles.focused)
+          ...styles.main
```

Comment:
now that there's only one style object being used here, this line can be simplified to:
```
style={styles.main}
```

Reply:
(same on lines 56-58)